pkg/chain/ethereum/connect.go

```diff
@@ -106,17 +108,28 @@ func Connect(
 
 	blockCounter, err := ethutil.NewBlockCounter(wrappedClient)
 	if err != nil {
-		return nil, nil, fmt.Errorf(
+		return nil, fmt.Errorf(
 			"failed to create Ethereum blockcounter: [%v]",
 			err,
 		)
 	}
 
+	tbtcSystemAddress, err := config.ContractAddress(
+		BondedECDSAKeepFactoryContractName,
```

Comment:
Shouldn't it be: 
```suggestion
		TBTCSystemContractName,
```
??

Reply:
Yepp, good catch 😅 I think I flubbed this when I was splitting the work into good commits.

Reply:
Fix't.